tests/integration/test_termination.py

```diff
@@ -2,33 +2,43 @@
 
 
 class TestCaseTermination(basetest.BaseTest):
-    def _assert_correct_shutdown(self, exitcode=0):
-        assert self.await_container_health("unhealthy", 60)
-        assert self.get_container_exitcode() == exitcode
-        self.assert_string_in_recent_logs("Mendix Runtime is shutting down")
-        self.assert_string_in_recent_logs("Mendix Runtime is now shut down")
-        if exitcode == 0:
-            # sys.exit(1) only occurs before the await termination loop
-            self.assert_string_in_recent_logs(
-                "Runtime process has been terminated"
-            )
-        self.assert_string_in_recent_logs("Terminating process group")
-
-    # Tests if termination works if a shutdown command is sent to the runtime
-    def test_termination_shutdown_command(self):
-        self.stage_container("Mendix8.1.1.58432_StarterApp.mda")
+    def _test_termination_signal(
+        self, signal="SIGTERM", env_vars={}, exitcode=0
```

Comment:
Let's have `env_vars` default to None, else we might run into the mutable default argument gotcha.